packages/xrpl/src/models/transactions/UNLModify.ts

```diff
@@ -0,0 +1,18 @@
+/**
+ * Mark a change to the Negative UNL.
+ *
+ * @category Pseudo Transaction Models
+ */
+export interface UNLModify {
+  TransactionType: 'UNLModify'
+  /**
+   * The ledger index where this pseudo-transaction appears. This distinguishes the pseudo-transaction from other occurrences of the same change.
+   */
+  LedgerSequence: number
+  /**
+   * If 1, this change represents adding a validator to the Negative UNL. If 0, this change represents removing a validator from the Negative UNL. (No other values are allowed.)
```

Comment:
I think the "No other types are allowed" is implied by the type definition, and if we flip the order of explanations then it'll match the type definition (which I think is especially helpful since this is a double-negative situation so it's easy to get tripped up trying to map it all out in your head) 
```suggestion
   * If 0, this change represents removing a validator from the Negative UNL. If 1, this change represents adding a validator to the Negative UNL.
```

Reply:
I copied these from xrpl.org.  I will clean up for this medium.